### ISSUE TITLE
Content Audit

### ISSUE THREAD

**Issue by raveling** [avelingrayorg/cc#2]
- Audit content from Caylia's TOC wishlist
- Acknowledge existing sites as part of that process
- Factor in high-level business objectives
- Integrate the audit with Core Model Approaches

**Comment by bobbymray** [comment]
I barely remember what Core Model is all about :(

We should hopefully Skype before meeting with C&S to Audit, Acknowledge, Factor, and Integrate our pitch.

and weird, external links seem to be blocked in wafflle - have you clicked the link to the google doc?

**Comment by raveling** [comment]
Yeah that link is Bum. I think it's a GH thing as I had same issue there a while back if I recall correctly. 

I think we should take a long hard look at what we've done so far for $250. I am at 16 hours, but i did spend at least 5 on that audit. My sense is we pitch a few options based on her budget and see where thing lie. Let's have a pre-meeting meeting fosho, maybe a few days before.
